pkgs/dartpad_ui/lib/editor/editor.dart

```diff
@@ -303,6 +311,8 @@ class _EditorWidgetState extends State<EditorWidget> implements EditorService {
   @override
   void dispose() {
     listener?.cancel();
+    _autosaveTimer?.cancel();
+    _autosave();
```

Comment:
Why is _autosave() called after the editor is disposed?

Reply:
As far as I'm aware, the editor itself isn't disposed at this point. The intention here was to do one last auto-save before closing the editor. But I couldn't get it to trigger in my tests so I removed it.